### ISSUE TITLE
Multiline labels

### ISSUE THREAD

**Issue by H2020** [gephi/gephi#787]
I suggest long labels appear on many lines on the graph.

**Comment by lobeg25** [comment]
Yes, that would be so appreciated. In GraphViz, you can add "\l", i.e. two characters '\' and 'l' to change line. Same would be great.

**Comment by rjurney** [comment]
This would be great. I need this.

**Comment by james-g** [comment]
I also vote for multi-line labels.

**Comment by marcel-goldschen-ohm** [comment]
Yup, I'd love this feature too ;)

**Comment by qcho** [comment]
absolute must!

**Comment by antonpetrashkevich** [comment]
Requesting!

**Comment by PeterAJansen** [comment]
I'm a new Gephi user and was also hoping this feature would be added when I started using Gephi about a year ago.  I recently came to a point where the feature was required (a very busy graph with very long labels), and dove into the 0.9.2 code.  It looks like one of the reasons this seemingly simple feature hasn't been added for several years is that it would require a large rearchitecting of the text rendering -- right now both the text storage class that interfaces with TextRenderer (which calculates the bounding boxes) assumes single lines of text, and all the rendering functions for node/edge labels across the different modules make this assumption too.  This is a moderate rearchitecting job that should be done by someone familiar with the project (and not a person from the community unfamiliar with the enormous code base, like me).  That being said, I downloaded a copy of IntelliJ and the latest source, and hacked my copy to do multiline node label rendering inside two renderers: the overview (when you're browsing the graph), and the Preview PDF exporter (but not the preview module preview itself, or any of the other exporters, although this would be easy to add by following the style).  It works right at the renderer, breaking any labels into multiple lines on "<br>" tags.  The modifications can be found here: https://github.com/gephi/gephi/compare/master...PeterAJansen:master . This obviously shouldn't be added to the official source tree since it's not a good/clean solution, but if someone else needs this feature in a pinch, I hope it helps.

**Comment by songjinghe** [comment]
I am badly in need of this feature in preview mode, there can be an option to switch wrap or not.

**Comment by maxholgasson** [comment]
is this available in the meantime?

**Comment by eduramiba** [comment]
This is not available yet

**Comment by visimagescode** [comment]
Expecting!

**Comment by blackappslist** [comment]
2020 - and Still Waiting......

**Comment by clermagic224** [comment]
Is this feature available now?

**Comment by uzaymd** [comment]
Very much needed!

**Comment by eltrompetero** [comment]
Seconding all the comments. Would be great if this could be done automatically without having to insert a newline character, i.e. each line is X characters and Y lines are shown.
